src/finite_elasticity_routines.f90

```diff
@@ -3597,10 +3598,16 @@ SUBROUTINE FINITE_ELASTICITY_GAUSS_CAUCHY_TENSOR(EQUATIONS_SET,DEPENDENT_INTERPO
 
 !      CALL Eigenvalue(C_e,EVALUES,ERR,ERROR,*999)
       CALL DSYEV('V','U',3,C_e,3,EVALUES,WORK,-1,ERR)
-      IF(ERR.NE.0) WRITE(*,*) "Error in Eigenvalue computation"
+      IF(ERR.NE.0) THEN
+        LOCAL_ERROR="Error in Eigenvalue computation"
+        CALL FlagError(LOCAL_ERROR,ERR,ERROR,*999)
+      END IF
```

Comment:
OK. You can also just do 

CALL FlagError("Error in Eigenvalue computation.",err,error,*999)